docs/bcfusion/developers/staking.md

```diff
@@ -134,7 +134,105 @@ function claimBatch(address[] calldata operatorAddresses, uint256[] calldata req
 - `operatorAddress`: The operator addresses of the validatores.
 - `requestNumber`: The numbers of unbonding requests to claim from the validators.
 
+## FAQs
+
+### What are the functions/interfaces of each validator's credit contract?
+
+For each validator, there is a credit contract which will be automatically deployed when it is created.
```

Comment:
this did not explain the `functions` of credit contract

Reply:
updated